usmqe/api/tendrlapi/common.py

```diff
@@ -0,0 +1,84 @@
+"""
+Tendrl REST API common functions.
+"""
+
+import time
+import pytest
+import requests
+from usmqe.api.base import ApiBase
+
+LOGGER = pytest.get_logger("commonapi", module=True)
+
+
+class ApiCommon(ApiBase):
```

Comment:
I understand why there is ApiCommon and ApiBase but maybe it would be better to come up with different names or something (like TendrlCommon, TendrlGluster, TendrlCeph for items in tendrlapi). It seems a little confusing. But it's just my opinion.